src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -747,6 +745,9 @@ getTxFee(Application const& app, Config const& config, Json::Value tx)
         if (!tx[jss::Signers].isArray())
             return config.FEES.reference_fee;
 
+        if (tx[jss::Signers].size() > STTx::maxMultiSigners(&ledger->rules()))
+            return config.FEES.reference_fee;
```

Comment:
Can we add a test for this line?

Reply:
Yeah I think throw makes more sense... I assume it'll throw at the local checks function anyways?

Reply:
Actually it is not caught, and there is a throw in getCurrentNetworkFee that will cause an unhandledException but its impossible to trigger without changing Tuning::.

Reply:
Perhaps it would be better if it returned whatever the fee would be for 32 signers, rather than just the reference fee - I feel like queue processing might get messed up otherwise (since that transaction would probably have a much higher fee level).

Reply:
Technically the transaction would fail in preflight for having too many signers right? This is just here because this RPC is calling calculateBaseFee before preflight. (This is the only location this happens)

Reply:
Perhaps I'm overthinking this... Yeah I guess preflight happens before any queue things